app/Models/User.php

```diff
@@ -154,4 +154,8 @@ public function profile() {
     public function nominations() {
         return $this->belongsToMany('App\Models\Nominee','vouches','voucher_id','nominee_id');
     }
+
+    public function isCoordinapeUser() {
```

Comment:
`public function getIsCoordinapeUserAttribute() {
    return $this->role == config('enums.user_types.coordinape');
}`

you could just do this, if you are using role to track whether its coordinape user